<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Form</class>
    <widget class="QWidget" name="Form">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>377</width>
                <height>455</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
                <widget class="QSplitter" name="splitter">
                    <property name="orientation">
                        <enum>Qt::Vertical</enum>
                    </property>
                    <widget class="QWidget" name="">
                        <layout class="QVBoxLayout" name="verticalLayout">
                            <item>
                                <widget class="QLabel" name="serverNameLabel">
                                    <property name="sizePolicy">
                                        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                            <horstretch>0</horstretch>
                                            <verstretch>0</verstretch>
                                        </sizepolicy>
                                    </property>
                                    <property name="text">
                                        <string>Server Name</string>
                                    </property>
                                    <property name="alignment">
                                        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QLineEdit" name="serverNameLine"/>
                            </item>
                            <item>
                                <spacer name="verticalSpacer">
                                    <property name="orientation">
                                        <enum>Qt::Vertical</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>20</width>
                                            <height>40</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                        </layout>
                    </widget>
                    <widget class="QWidget" name="">
                        <layout class="QHBoxLayout" name="horizontalLayout">
                            <item>
                                <widget class="QLabel" name="label_2">
                                    <property name="text">
                                        <string>Width in Blocks</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <spacer name="horizontalSpacer">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>40</width>
                                            <height>20</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                            <item>
                                <widget class="QSpinBox" name="widthBox">
                                    <property name="minimum">
                                        <number>10</number>
                                    </property>
                                    <property name="maximum">
                                        <number>100</number>
                                    </property>
                                    <property name="value">
                                        <number>40</number>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </widget>
                    <widget class="QWidget" name="">
                        <layout class="QHBoxLayout" name="horizontalLayout_2">
                            <item>
                                <widget class="QLabel" name="label_3">
                                    <property name="text">
                                        <string>Height in Blocks</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <spacer name="horizontalSpacer_2">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>40</width>
                                            <height>20</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                            <item>
                                <widget class="QSpinBox" name="heightBox">
                                    <property name="minimum">
                                        <number>10</number>
                                    </property>
                                    <property name="maximum">
                                        <number>100</number>
                                    </property>
                                    <property name="value">
                                        <number>30</number>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </widget>
                    <widget class="QWidget" name="">
                        <layout class="QHBoxLayout" name="horizontalLayout_3">
                            <item>
                                <widget class="QLabel" name="label_4">
                                    <property name="text">
                                        <string>Static Food Amount</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <spacer name="horizontalSpacer_3">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>40</width>
                                            <height>20</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                            <item>
                                <widget class="QSpinBox" name="foodBox">
                                    <property name="maximum">
                                        <number>100</number>
                                    </property>
                                    <property name="value">
                                        <number>1</number>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </widget>
                    <widget class="QWidget" name="">
                        <layout class="QHBoxLayout" name="horizontalLayout_4">
                            <item>
                                <widget class="QLabel" name="label_5">
                                    <property name="text">
                                        <string>Step Delay</string>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <spacer name="horizontalSpacer_4">
                                    <property name="orientation">
                                        <enum>Qt::Horizontal</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>40</width>
                                            <height>20</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                            <item>
                                <widget class="QSpinBox" name="delayBox">
                                    <property name="minimum">
                                        <number>100</number>
                                    </property>
                                    <property name="maximum">
                                        <number>3000</number>
                                    </property>
                                    <property name="singleStep">
                                        <number>100</number>
                                    </property>
                                    <property name="value">
                                        <number>1000</number>
                                    </property>
                                </widget>
                            </item>
                        </layout>
                    </widget>
                    <widget class="QPushButton" name="startButton">
                        <property name="text">
                            <string>START</string>
                        </property>
                    </widget>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
